tests/test_bug_features.py

```diff
@@ -38,15 +39,26 @@
 
 @pytest.fixture
 def read(get_fixture_path):
-    def _read(path, feature_extractor_class, expected_results):
+    def _read(
+        path,
+        feature_extractor_class,
+        expected_results,
+        use_inline_data=False,
+        inline_data=None,
```

Comment:
Why do we have both `use_inline_data` and `inline_data`? Would it be more straightforward to rely on `inline_data` only?

Reply:
63672d60d650acf49bf1bbc3863ee133b7a55512